### ISSUE TITLE
Metadata area in a component should have a heading and landmark

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/pulfalight#1777]
refs #1717 
[full list of issues for audit1717](https://github.com/pulibrary/pulfalight/issues?q=is%3Aissue%20state%3Aopen%20audit1717)

### Location of problem on site
component page, especially one with access restrictions https://findingaids.princeton.edu/catalog/C0108_c3741

### Description of error / warning
https://wave.webaim.org/api/docs?format=html#heading_missing

When a component page has an access restriction, it has an h3. But below that there's not a new h3 to indicate the metadata area / component description. It's not clear that this is really a different type of content.

In general when navigating with VoiceOver, the section of the page that contains the component description is easy to lose. The "main content" area has all kinds of other things in it as well. It should at least have an aria label around it so it shows up in the rotor landmarks menu.

### WCAG success criteria

1.3.1 Info and Relationships (Level A)
2.4.6 Headings and Labels (Level AA)

### WCAG conformance level

A

### Recommended fix

- add a heading to the metadata area, maybe a screenreader-only heading
- use an aria-label around the metadata area

### Tool used to find the problem

wave and VoiceOver

### Screenshot

<img width="837" height="589" alt="Image" src="https://github.com/user-attachments/assets/0ebb9331-24ff-46a0-b165-6aa24bdc8610" />
